authenticator_sdk/src/main/kotlin/com/saltedge/authenticator/sdk/web/SEFrameWebView.kt

```diff
@@ -46,6 +47,14 @@ open class SEFrameWebView : WebView {
         setupView()
     }
 
+    override fun setOverScrollMode(mode: Int) {
+        runCatching {
+            super.setOverScrollMode(mode)
```

Comment:
Please simulate an exception in SDK and prove that it has been logged by crashlytics.